lib/chat/admin_db_structures.livemd

```diff
@@ -1,5 +1,69 @@
 # AdminDB structures
 
+## Admin room
+
+### Room
+
+Should be unique.
+
+<!-- livebook:{"force_markdown":true} -->
+
+```elixir
+{:pub_key, admin_room_pub_key}
+```
+
+<!-- livebook:{"break_markdown":true} -->
+
+### Admin
+
+Admins are the users that visited admin room once at least. New potential admins can be invited from admin panel.
+
+<!-- livebook:{"force_markdown":true} -->
+
+```elixir
+{{:new_admin, admin_pub_key}, admin_card}
+```
+
+<!-- livebook:{"break_markdown":true} -->
+
+### Wifi password
+
+<!-- livebook:{"force_markdown":true} -->
+
+```elixir
+{:wifi_password, hashed_password}
+```
+
+<!-- livebook:{"break_markdown":true} -->
+
+### Cargo settings
+
+Set cargo checkpoints. Checkpoints are automatically invited to the Cargo rooms.
+
+<!-- livebook:{"force_markdown":true} -->
+
+```elixir
+{:cargo_settings, CargoSettings{checkpoints: [pub_key1, pub_key2, pub_key3]}}
+```
+
+<!-- livebook:{"break_markdown":true} -->
+
+### Media settings
+
+When new USB drive is plugged into the secondary port,the following functionality will be started.
```

Comment:
When empty USB drive ....
Functionalities: 
 - backup - full backup of main drive
 - onliners - backup of data in scope of online (at this moment) users
 - cargo - special cargo scenario